lib/board.rb

```diff
@@ -0,0 +1,75 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells
+
+  def initialize(cells = nil)
+    @cells = cells || Array.new(9, " ")
+  end
+
+  def input_to_index(user_input)
+    user_input.to_i - 1
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+    puts
+    puts 
+    puts
+  end
+
+  def position(move)
+    move = input_to_index(move)
+    @cells[move]
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def full?
+    @cells.include?(" ") == false
+  end
+
+  def turn_count
```

Comment:
Use the highest level and most expressive iterator possible to solve a problem. [`count`](http://ruby-doc.org/core-2.2.0/Array.html#method-i-count) on the cells is perfect and the rest of this method is not needed. Entire body could read:

```ruby
def turn_count
  cells.count { |cell| cell == "X" || cell == "O" }
end
```